src/applications/vaos/services/appointment/transformers.js

```diff
@@ -402,7 +403,22 @@ function setContained(appt) {
           ],
         };
 
-        if (appt.sta6aid) {
+        if (tasInfo) {
+          service.location = {
```

Comment:
I should have caught this in @narin's suggested change, but the `location` field on `HealthcareService` resources is a [reference](http://hl7.org/fhir/dstu2/references.html), which means it doesn't have the actual location info here, just a field with an id. 

I think the way to handle this is to create a `Location` resource object and add it to the `contained` array in here.

You can see an example of transforming data into the Location resource [here](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/vaos/services/location/transformers.js#L159), though you only need to set `resourceType` and `address`, I think.